pkg/project/v2/project_loader.go

```diff
@@ -191,22 +189,46 @@ func loadProject(fs afero.Fs, context ProjectLoaderContext, projectDefinition ma
 		}
 	}
 
-	// The scope parameter of a key user actions web configuration needs to be a reference to another application-web config
-	// The reference parameter makes sure that rely on the fact that kua web configs are loaded/deployed within the same
-	// sub graph (independent component) later on as
 	for _, c := range configs {
+		// The scope parameter of a key user actions web configuration needs to be a reference to another application-web config
+		// The reference parameter makes sure that rely on the fact that kua web configs are loaded/deployed within the same
+		// sub graph (independent component) later on as
 		if c.Coordinate.Type == api.KeyUserActionsWeb {
 			if _, ok := c.Parameters[config.ScopeParameter].(*ref.ReferenceParameter); !ok {
 				errors = append(errors, fmt.Errorf("scope parameter of config of type '%s' with ID '%s' needs to be a reference "+
 					"parameter to another web-application config", api.KeyUserActionsWeb, c.Coordinate.ConfigId))
 			}
 		}
 	}
-
 	if errors != nil {
 		return Project{}, errors
 	}
 
+	// Network zone API configs have a hard dependency on the "builtin:networkzones" setting 2.0 config
```

Comment:
Please describe WHY we actually need to do this, currently this extensive doc comment mainly tells me that there's some "dependency" and WHAT the code below does.

What I get from the code and variable naming, the setting is for (de)activating networkzones in general, so it needs to be configured for any zones (as their API calls would fail, if the setting is off I assume?)? 
If that's correct, the code actually tells me more about the WHY than 3 lines of comments ;)